fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/database/entities/LoginEntity.java

```diff
@@ -18,8 +15,10 @@
 @Builder
 public class LoginEntity {
     @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "login_generator")
+    @SequenceGenerator(name = "login_generator", sequenceName = "login_id_seq", allocationSize = 1)
```

Comment:
Using allocationSize 1 is a bad idea from a performance perspective as it is an entity-per-request class. Use 50 as sane default. See below

Reply:
done